virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -360,135 +358,144 @@ include::snippets/technology-preview.adoc[]
 |Element
 |Description
 
-|*Volumes project*
-|The current namespace.
-
-|*Select volume to boot from*
-|Select a volume from the list to boot your virtual machine from.
+|*Volumes project* field
+|Project in which bootable volumes are stored. The default is `openshift-virtualization-os-images`.
 
-|*Add volume*
-|Upload a new volume or use an existing PersistentVolumeClaim.
+|*Add volume* button
+|Click to upload a new volume or to use an existing persistent volume claim.
 
 |*Filter* field
-|Filter boot sources by operating system.
+|Filter boot sources by operating system or resource.
 
 |Search field
-|Search boot sources by keyword.
+|Search boot sources by name.
 
-|*Select InstanceType*
-|Select an instance type based on the number of CPUs and amount of memory available.
+|Volume table
+|Select a bootable volume for your virtual machine.
+
+|InstanceType tiles
+|Select an instance type for your virtual machine.
 
 |*VirtualMachine details* pane
-|Displays the current virtual machine settings. You can change the name of the virtual machine during this step.
+|Displays the virtual machine settings.
+
+|*Name* field
+|Optional: Enter the virtual machine name.
+
+|*SSH key name*
+|Click the edit icon to add a public SSH key to create a secret or to select an existing secret.
```

Comment:
This sentence feels unwieldy to me. Can it be simplified or divided into two sentences?